runtime/src/main/protobuf/google/api/http.proto

```diff
@@ -0,0 +1,376 @@
+// Copyright 2019 Google LLC.
```

Comment:
This file is not needed for server reflection

Reply:
it is needed to be able to process annotations.proto (https://github.com/akka/akka-grpc/pull/654/files#diff-bd58959d0ef01f941f2e2a00d2a788baR30)

Reply:
Yeah, but I don't think you need annotations.proto, I only included it in Cloudstate because it is a builtin feature of Cloudstate.

Reply:
Aaah I didn't realize, will remove it then!